### PR TITLE
Updates to event notifications docs

### PR DESCRIPTION
**Pull request by jonesphillip**
Updates to event notifications docs, also removed label for Sippy in Wrangler command docs

### CODE REVIEW

File: src/content/docs/r2/buckets/event-notifications.mdx [hunk 1]
```diff
@@ -224,12 +229,7 @@ Queue consumers receive notifications as [Messages](/queues/configuration/javasc
...
-During the beta, event notifications has the following limitations:
+## Notes
 
-- Queues [per-queue message throughput](/queues/platform/limits/) is currently 400 messages per second. If your workload produces more than 400 notifications per second, messages may be dropped.
-- For a given bucket, only one event notification rule can be created per queue.
-- Each bucket can have up to 5 event notification rules.
-- Deletes that occur as a result of object lifecycle policies will not trigger an event notification.
-- Event notifications are not available for buckets with [jursdictional restrictions](/r2/reference/data-location/#jurisdictional-restrictions).
+- Event notifications are not available for buckets with [jursdictional restrictions](/r2/reference/data-location/#jurisdictional-restrictions) (_coming soon_).
+- Queues [per-queue message throughput](/queues/platform/limits/) is currently 5,000 messages per second. If your workload produces more than 5,000 notifications per second, we recommend splitting notification rules across multiple queues.
```
Comment (irvinebroque) [inline src/content/docs/r2/buckets/event-notifications.mdx:235]: Is there something in the Queues docs that shows you how to do this, that we should link to?

**Review by irvinebroque — APPROVED** [review]

File: src/content/docs/r2/buckets/event-notifications.mdx [hunk 2]
```diff
@@ -5,12 +5,6 @@ pcx_content_type: how-to
 
 Event notifications send messages to your [queue](/queues/) when data in your R2 bucket changes. You can consume these messages with a [consumer Worker](/queues/reference/how-queues-works/#create-a-consumer-worker) or [pull over HTTP](/queues/configuration/pull-consumers/) from outside of Cloudflare Workers.
 
-:::note[Open Beta]
-
-The event notifications feature is currently in open beta. To report bugs or request features, go to the #r2-storage channel in the [Cloudflare Developer Discord](https://discord.cloudflare.com) or fill out the [feedback form](https://forms.gle/2HBKD9zG9PFiU4v79).
-
-:::
-
 ## Get started with event notifications
```
Comment (dcpena) [inline src/content/docs/r2/buckets/event-notifications.mdx:8]: How do you feel about deleting this header and then making Prerequisites, and the two headers that follow, an H2 instead?

File: src/content/docs/r2/buckets/event-notifications.mdx [hunk 3]
```diff
@@ -21,13 +15,23 @@ Before getting started, you will need:
 - An existing queue. If you do not already have a queue, refer to [Create a queue](/queues/get-started/#3-create-a-queue).
 - A [consumer Worker](/queues/reference/how-queues-works/#create-a-consumer-worker) or [HTTP pull](/queues/configuration/pull-consumers/) enabled on your Queue.
 
-### Set up Wrangler
+### Enable event notifications via Dashboard
+
+1. From the Cloudflare dashboard, select **R2** from the sidebar.
+2. Select the bucket you'd like to add an event notification rule to.
+3. Switch to the **Settings** tab, then scroll down to the **Event notifications** card.
+4. Select **Add notification** and choose the queue you'd like to receive notifications and the [type of event(s)](/r2/buckets/event-notifications/#event-types) that will trigger them.
```
Comment (dcpena) [inline src/content/docs/r2/buckets/event-notifications.mdx]:
```suggestion
4. Select **Add notification** and choose the queue you'd like to receive notifications and the [type of event](/r2/buckets/event-notifications/#event-types) that will trigger them.
```

Comment (dcpena) [reply src/content/docs/r2/buckets/event-notifications.mdx]: Switched to singular but plural would also be fine. Ultimately, just removing the `(s)`.

**Review by dcpena — APPROVED** [review]

**Pull request merged**